wandb/sdk/artifacts/artifact.py

```diff
@@ -1846,8 +1844,6 @@ def _download_using_core(
         )
         result = handle.wait(timeout=-1)
 
-        if result is None:
```

Comment:
will it be handled by the close logic?

Reply:
`result` can only be `None` on timeout, in which case abandoning the handle is unnecessary

Reply:
oh i see thanks!